ProjectLighthouse.Servers.Website/Controllers/AutoDiscoverController.cs

```diff
@@ -0,0 +1,26 @@
+using LBPUnion.ProjectLighthouse.Configuration;
+using LBPUnion.ProjectLighthouse.Servers.Website.Types;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LBPUnion.ProjectLighthouse.Servers.Website.Controllers;
+
+[ApiController]
+public class AutoDiscoverController: ControllerBase
+{
+    [ResponseCache(Duration = 86400)]
+    [HttpGet("/autodiscover")]
```

Comment:
```suggestion
    [HttpGet("autodiscover")]
```
Doesn't affect functionality but standard is to leave off the leading slash